freedomofpress/securedrop-client
# remove user refresh and replace with sync icon

# Description

Closes https://github.com/freedomofpress/securedrop-client/issues/687

We now animate the metadata sync and instead of showing an active state for the entire length of the metadata sync we always just show a 2 second animation. Here is recording of the animation (it's subtle) after I changed the background metadata syncs to happen every 6 seconds for demo purposes:

![metadatsync](https://user-images.githubusercontent.com/4522213/73114161-00271300-3ecd-11ea-8738-98d1166883f9.gif)

# Test Plan

1. Start the client in offline mode and check that icon is offline icon
2. Log in and see the metadata sync animation
3. After 2 seconds see that it is in the inactive state
4. Log out see the icon switches back to offline mode
